### PR TITLE
ask user to read licnese before installation

### LINKED ISSUE #8: Before performing `saspy` install, ask for compliance with license
To be careful with the complicated `saspy` license https://github.com/sassoftware/saspy/blob/main/LICENSE.md, let's please ask the user to read that and confirm that they comply with this license before actually performing the installation.

### PR DESCRIPTION
**Pull request by clarkliming**
close #8

### CODE REVIEW

**Comment by clarkliming** [comment]
hi @cicdguy could you please help check why these checks are complaining? I have no idea why it could not find package `desc` or `lintr`

**Comment by clarkliming** [comment]
thank you @cicdguy I removed the renv related things

File: R/utils.R [hunk 1]
```diff
@@ -1,23 +1,45 @@
...
+#' @param method (`character`)\cr method to install `saspy`.
+#' @param conda (`character`)\cr path to `conda` executable.
+#'
 #' @export
 install_saspy <- function(method = "auto", conda = "auto") {
+  msg <- paste0(
+    "Before installing saspy, please read and confirm that you will ",
+    "comply to the lincense of saspy:\n",
+    "https://github.com/sassoftware/saspy/blob/main/LICENSE.md\n",
+    "I have read the license and confirm that I will comply to the lincense:"
```
Comment (danielinteractive) [inline R/utils.R:12]:
```suggestion
    "I have read the license and confirm that I will comply to the license:"
```

**Review by danielinteractive — APPROVED** [review]
Nice cleanup and feature! Thanks @clarkliming

**Pull request merged**